android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -74,6 +77,8 @@ class PlayerModule(context: ReactApplicationContext) : BitmovinBaseModule(contex
         val playerConfig = playerConfigJson?.toPlayerConfig() ?: PlayerConfig()
         val analyticsConfig = analyticsConfigJson?.toAnalyticsConfig()
         val defaultMetadata = analyticsConfigJson?.getMap("defaultMetadata")?.toAnalyticsDefaultMetadata()
+        val enableMediaSession = playerConfigJson?.getMap("mediaControlConfig")
+            ?.toMediaControlConfig()?.isEnabled ?: true
```

Comment:
Nit: I would prefer defaulting to `false` here as this is rather an error case (I assume a `mediaControlConfig` should always be present here?). The correct default value should be handled somewhere else I guess.

Reply:
I'd prefer to keep it this way since a `mediaControlConfig` may not be present at all. Because it is a ReactNative-only object, so Native SDKs are not setting this explicitly